src/codecs/gif.rs

```diff
@@ -289,10 +289,29 @@ impl<'a, R: Read + 'a> AnimationDecoder<'a> for GifDecoder<R> {
     }
 }
 
+/// Number of repetitions for a GIF animation
+#[derive(Copy, Clone)]
+pub enum Repeat {
+    /// Finite number of repetitions
+    Finite(u16),
+    /// Looping GIF
+    Infinite,
+}
+
+impl From<Repeat> for gif::Repeat {
+    fn from(repeat: Repeat) -> Self {
+        match repeat {
+            Repeat::Finite(n) => gif::Repeat::Finite(n),
+            Repeat::Infinite => gif::Repeat::Infinite,
+        }
+    }
+}
```

Comment:
Including this conversion would make the exact version of `gif` a public dependency. Don't add it. (It actually makes CI fail, that's at least reassuring to see).

Reply:
Ah yes, did not know this was one of your policies. I originally wanted to straight up use (and re-export) the `gif::Repeat`  enum, but it does not implement `Copy` nor `Clone`, which is kind of a pain. Changing this!

Reply:
Updated. CI seems happy now, but I'll let you be the last judge on this.